source/MRMesh/MRDistanceMapTests.cpp

```diff
@@ -289,8 +292,9 @@ TEST( MRMesh, DistanceMapNegativeValue )
     const bool saveMesh = true;
     if ( saveMesh )
     {
-        MeshSave::toMrmesh( distanceMapToMesh( dm, params ), std::filesystem::path( "c:/temp/dm.mrmesh" ) );
-        MeshSave::toMrmesh( distanceMapToMesh( dm2, params2 ), std::filesystem::path( "c:/temp/dm2.mrmesh" ) );
+        // without a progress bar, distanceMapToMesh cannot return an error
```

Comment:
Now it cannot, but later some more errors can be introduced

Reply:
https://github.com/MeshInspector/MeshInspectorCode/issues/3896